cmd/containerd-shim/main_unix.go

```diff
@@ -41,6 +41,7 @@ import (
 	shimapi "github.com/containerd/containerd/runtime/v1/shim/v1"
 	"github.com/containerd/ttrpc"
 	"github.com/containerd/typeurl"
+	"github.com/docker/docker/pkg/reexec"
```

Comment:
Using this as a dependency is a big no no. Either reimplement or find another solution.

DRY is a liability here.